proposals/NNNN-Persisting HMI Capabilities specific to headunit.md

```diff
@@ -0,0 +1,89 @@
+# Persisting HMI Capabilities specific to headunit
+
+* Proposal: [SDL_NNNN](NNNN-Persisting HMI Capabilities specific to headunit.md)
+* Author: [Ankur Tiwari](https://github.com/atiwari9)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core]
```

Comment:
Should `HMI` be included as an Impacted Platform as well?

Reply:
No, no changes are needed for HMI